workflows/scripts/aggregate_cells.py

```diff
@@ -95,11 +95,11 @@ def aggregate(
         )
 
     logging.info(f"Identified plate map: {platemap}")
-    platemap_file = os.path.join(metadata_dir, "platemap", "{}.csv".format(platemap))
+    platemap_file = pathlib.Path(metadata_dir) / "platemap" / f"{platemap}.csv"
```

Comment:
Consider using an f-string within the `pathlib.Path` creation to remove the need for extra spaces and quotations to form this path. I can also see why you might want to leave things as-is, so feel free to ignore this comment.